Pasing from SAS to R second program

```diff
@@ -0,0 +1,102 @@
+Moving Second Program from SAS to R
+
+This are the notes that apper as I move forward. I have the code in a separate document. 
+
+
+line 28 of SAS, I don't know why are we deleting the data that is NOT 1. 
+
+
+To beging the move to R, I'm going to import the following 3 datasets from SAS
+possiblenclass
```

Comment:
I don't mind a few typos from time to time, but you have "begin" as "beging" repeatedly as well as "first" as "firts" throughout this file.